apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffling.java

```diff
@@ -278,7 +281,7 @@ public static int getHoldingShufflingCount(long holdingId, boolean includeFinish
     static void addShuffling(Transaction transaction, ShufflingCreation attachment) {
         Shuffling shuffling = new Shuffling(transaction, attachment);
         insert(shuffling);
-        ShufflingParticipant.addParticipant(shuffling.getId(), transaction.getSenderId(), 0);
+        shufflingService.addParticipant(shuffling.getId(), transaction.getSenderId(), 0);
```

Comment:
1. I think **shuffling entity + dao/table** should be extracted and moved to separate components from that class.
They both still exist in that big component.

2. In general that I think
`public final class Shuffling`
should be refactored to 'ShufflingService' component.

Reply:
I did big refactoring part for Shuffling, so we will see that in the next PR.